nflow-explorer/src/index.html

```diff
@@ -60,7 +46,6 @@
     <script src="bower_components/dagre-d3/dist/dagre-d3.core.js"></script>
     <script src="bower_components/chart.js/dist/Chart.js"></script>
     <script src="bower_components/angular-chart.js/dist/angular-chart.js"></script>
-    <script src="bower_components/svg-pan-zoom/dist/svg-pan-zoom.js"></script>
```

Comment:
Why is this removed? Though not superfatal, but removing this line breaks zooming for large workflows, and you always get errors to browser console. Example of large workflow is in nflow-tests (wordGenerator workflow). It's easily started by:
```
java -jar nflow-tests/target/nflow-tests-7.4.1-SNAPSHOT.jar
```

Reply:
should not be, it is just that it is automatically removed from there and I already restored it 2 times. but latest commit again removed it...